includes/GlobalFunctions.php

```diff
@@ -171,18 +171,19 @@ function &smwfGetSparqlDatabase() {
  * Compatibility helper for using Linker methods.
  * MW 1.16 has a Linker with non-static methods,
  * where in MW 1.19 they are static, and a DummyLinker
- * class is introduced, which can be instantaited for
- * compat reasons.
+ * class is introduced, which can be instantiated for
+ * compat reasons. As of MW 1.28, DummyLinker is being
+ * deprecated, so always use Linker.
```

Comment:
So now the comment says we're doing A because foo but actually do B because bar?